ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -941,105 +946,50 @@ def test021_create_existing_order_with_test_units(self):
-        article = self.order_page.get_article()
-        self.assertEqual('Search', article)
-        test_unit = self.order_page.get_test_unit()
-        self.assertEqual('Search', article)
-        self.order_page.set_article(article='a')
-        self.order_page.set_test_unit(test_unit=test_unit_dict['Test Unit Name'])
-
-        article = self.order_page.get_article()
-        self.order_page.save(save_btn='order:save_btn')
-        self.base_selenium.LOGGER.info(' + Order created with no : {} '.format(created_existing_order))
-        self.analyses_page.get_analyses_page()
-        self.base_selenium.LOGGER.info(
-            'Assert There is an analysis for this new order.')
-        orders_analyess = self.analyses_page.search(created_order)
-        latest_order_data = self.base_selenium.get_row_cells_dict_related_to_header(
-            row=orders_analyess[0])
-        self.assertEqual(
-            created_order.replace("'", ""), latest_order_data['Order No.'].replace("'", ""))
-        self.assertEqual(
-            article.split(' No:')[0], latest_order_data['Article Name'])
-        self.assertEqual(
-            'Subassembely', latest_order_data['Material Type'])
-        
-    # will continue with us
-    def test023_create_existing_order_with_test_units_and_change_article(self):
-        """
-        New: Orders with test units: Create a new order from an existing order with test units but change the article
+        New: Orders with test units: Create a new order from an existing order with
+        test units but change the material type or the article
 
         LIMS-3269
         """
-        self.base_selenium.LOGGER.info(
-            'Running test case to create an existing order with test units and change article')
-        test_units_list = []
-        test_unit_dict = self.get_active_tst_unit_with_material_type(search='Qualitative', material_type='All')
-        if test_unit_dict:
-            self.base_selenium.LOGGER.info('Retrieved test unit ' + test_unit_dict['Test Unit Name'])
-            test_units_list.append(test_unit_dict['Test Unit Name'])
+        order_no = self.order_page.create_existing_order_with_auto_fill()
+        self.order_page.sleep_tiny()
+        if edit_feild == 'material_type':
+            material_type = 'Subassembely'
+            self.order_page.set_material_type(material_type=material_type)
+            self.assertEqual(self.base_selenium.get_value(element='order:article'), None)
+            self.assertEqual(self.base_selenium.get_value(element='order:test_unit'), None)
+            article = self.order_page.set_article()
+            test_unit = self.order_page.set_test_unit()
+        else:
+            test_unit = self.order_page.get_test_unit()
+            material_type = self.order_page.get_material_type()
+            article = self.order_page.set_article()
+            # until bug in https://modeso.atlassian.net/browse/LIMSA-116 solved
```

Comment:
skip this test case with this bug

Reply:
I didn't skiped it as first part of material type works

Reply:
if there is any failer assertion in the test code, we should skip the whole test case regarding to this bug